pkg/archive/jar_test.go

```diff
@@ -312,3 +312,151 @@ func getSeparatedValues(attributeValue string) []string {
 	commaSeparateValues := strings.Split(value, ",")
 	return commaSeparateValues
 }
+
+func TestStableGitProperties(t *testing.T) {
+	testCases := []struct {
+		test     string
+		input    []*ZipEntry
+		expected []*ZipEntry
+	}{
+		{
+			test: "delete_git_properties",
+			input: []*ZipEntry{
+				{
+					&zip.FileHeader{Name: "META-INF/MANIFEST.MF"},
+					[]byte("Built-By: aman\r\n\r\n"),
+				},
+				{
+					&zip.FileHeader{Name: "git.properties"},
+					[]byte("git.build.user.email=mannu.poski10@gmail.com\r\ngit.build.user.name=Aman Sharma\r\n\r\n"),
+				},
+			},
+			expected: []*ZipEntry{
+				{
+					&zip.FileHeader{Name: "META-INF/MANIFEST.MF"},
+					[]byte("Built-By: aman\r\n\r\n"),
+				},
+			},
+		},
+		{
+			test: "delete_git_json",
+			input: []*ZipEntry{
+				{
+					&zip.FileHeader{Name: "META-INF/MANIFEST.MF"},
+					[]byte("Built-By: aman\r\n\r\n"),
+				},
+				{
+					&zip.FileHeader{Name: "git.json"},
+					[]byte("{\n" +
```

Comment:
we have "textwrap.Dedent" elsewhere in the repo which should make this easier to define with formatting inline.